packages/core/src/events.ts

```diff
@@ -72,10 +72,8 @@ class ClientRegister<T> implements ds.ClientRegister<T> {
     }
 
     emit(newValue: T) {
-        if (this.value !== newValue) {
```

Comment:
was there a problem with this?

Reply:
it's a bit misleading because in the case of axes ({x, y}), the equality check is not really useful.

Reply:
sure, but this is generic - what if people use for say booleans?

Reply:
[0x355724dd](https://github.com/microsoft/devicescript/commit/8cfdbcb4a297400dd8f07318ec10782909dbb868)